[2024-01-20T02:23:37.703+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bancodedados.insert_data manual__2024-01-20T02:23:35.381950+00:00 [queued]>
[2024-01-20T02:23:37.708+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bancodedados.insert_data manual__2024-01-20T02:23:35.381950+00:00 [queued]>
[2024-01-20T02:23:37.709+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-01-20T02:23:37.709+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-01-20T02:23:37.709+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-01-20T02:23:37.716+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): insert_data> on 2024-01-20 02:23:35.381950+00:00
[2024-01-20T02:23:37.720+0000] {standard_task_runner.py:55} INFO - Started process 49763 to run task
[2024-01-20T02:23:37.723+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'bancodedados', 'insert_data', 'manual__2024-01-20T02:23:35.381950+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/23-bancodedados.py', '--cfg-path', '/tmp/tmph60anvfo']
[2024-01-20T02:23:37.724+0000] {standard_task_runner.py:83} INFO - Job 126: Subtask insert_data
[2024-01-20T02:23:37.762+0000] {task_command.py:388} INFO - Running <TaskInstance: bancodedados.insert_data manual__2024-01-20T02:23:35.381950+00:00 [running]> on host cb9489d15b76
[2024-01-20T02:23:37.805+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bancodedados
AIRFLOW_CTX_TASK_ID=insert_data
AIRFLOW_CTX_EXECUTION_DATE=2024-01-20T02:23:35.381950+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-20T02:23:35.381950+00:00
[2024-01-20T02:23:37.806+0000] {sql.py:254} INFO - Executing: insert into teste values(1);
[2024-01-20T02:23:37.811+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2024-01-20T02:23:37.980+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2024-01-20T02:23:37.983+0000] {sql.py:375} INFO - Running statement: insert into teste values(1);, parameters: None
[2024-01-20T02:23:37.984+0000] {sql.py:384} INFO - Rows affected: 1
[2024-01-20T02:23:37.991+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=bancodedados, task_id=insert_data, execution_date=20240120T022335, start_date=20240120T022337, end_date=20240120T022337
[2024-01-20T02:23:38.024+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-01-20T02:23:38.043+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
